imports/ui/components/Card.jsx

```diff
@@ -110,22 +103,8 @@ class CardComponent extends Component {
               </div>
             )}
           </div>
-          {this.props.currentUser && (
-            <div>
-              <IconButton
-                style={{ display: this.state.isFav ? "none" : "" }}
-                onClick={this.onFavPressed}
-              >
-                <AddIcon />
-              </IconButton>
-              <IconButton
-                style={{ display: this.state.isFav ? "" : "none" }}
-                onClick={this.onFavPressed}
-              >
-                <CheckedIcon />
-              </IconButton>
-            </div>
-          )}
+          {/* Favorite icon is a part of the stretch goal to add wishlist, use later */}
+          {this.props.currentUser && <AddShoppingList item={this.state.data} />}
```

Comment:
Add to shopping list could add item multiple times when google between + icon and ✔️icon.  Also going from + to ✔️  doesn't remove the item from the shopping cart list. Do we want a way to remove items from shopping cart?

Reply:
#76